File: src/main/java/com/cuupa/dms/storage/tag/db/TagDatabaseSequence.java
package com.cuupa.dms.storage.tag.db;

import org.springframework.data.annotation.Id;
import org.springframework.data.mongodb.core.mapping.Document;

@Document(collection = "tag_database_sequence")
public class TagDatabaseSequence {

    @Id
    private String id;

    private long seq;

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public long getSeq() {
        return seq;
    }

    public void setSeq(long seq) {
        this.seq = seq;
    }
}

File: src/main/java/com/cuupa/dms/storage/tag/db/TagDatabaseSequence.kt
package com.cuupa.dms.storage.tag.db

import org.springframework.data.annotation.Id
import org.springframework.data.mongodb.core.mapping.Document

@Document(collection = "tag_database_sequence")
class TagDatabaseSequence {
    @Id
    var id: String? = null
    var seq: Long = 0

}